lib/api_repo.js

```diff
@@ -231,14 +231,22 @@ ApiRepo.prototype.buildPackages =
                 cleanName.replace(new RegExp('-', 'g'), '/');
             opts.packageInfo.api.name = that.pkgPrefix + cleanName;
             opts.packageInfo.api.version = version;
-            opts.packageInfo.api.buildCommonProtos =
-                that.opts.buildCommonProtos;
```

Comment:
This was only used by build-alt.gradle.mustache before - it was introduced by my change https://github.com/googleapis/packman/commit/d0209d8d45d24d16547aadfb502dc9b9b78de054 - and now it's not used anywhere any more.